tpot/tpot.py

```diff
@@ -1657,13 +1657,16 @@ def _combined_selection_operator(self, individuals, k):
         """
         return tools.selNSGA2(individuals, int(k / 5.)) * 5
 
-    def _random_mutation_operator(self, individual):
+    def _random_mutation_operator(self, individual, mutation_type=None):
```

Comment:
I understand. I'd rather lose coverage than modify the function to accommodate a unit test.